app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from typing import Callable
+
+
+class CommandError(Exception):
+    """Checks command input for right format."""
+
+
+def check_command(func: Callable) -> Callable:
+    def wrapper(command: str) -> Callable:
```

Comment:
You could use a decorator from `functools` to preserve the docstring and naming of a wrapped function.